src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -165,6 +182,10 @@ foam.CLASS({
     }
   `,
 
+  messages: [
+    { name: 'SUPPORT_TITLE', message: 'Payment Support: '}
```

Comment:
'Payment' is nanopay specific and if required, will have to be provided from nanopay code.